app/controllers/events/view/export.js

```diff
@@ -0,0 +1,53 @@
+import Ember from 'ember';
+
+const { Controller, run } = Ember;
+export default Controller.extend({
+  eventExportStatus  : '',
+  eventDownloadUrl   : '',
+  isDownloadDisabled : true,
+  image              : false,
+  audio              : false,
```

Comment:
Instead of this, do 

```
data: {
  image: false,
  audio: false,
  video: false,
  document: false
}
```

Now you can just pass the data property into the component